ckan/config/environment.py

```diff
@@ -146,6 +152,10 @@ def update_config() -> None:
         msg = "Invalid configuration values provided:\n" + msg
         raise CkanConfigurationException(msg)
 
+    # storages rely only on valid configuration
```

Comment:
Does this mean that config options from storages (e.g FsStorage's `path` or `create_path` won't be validated as validation happened a few lines before?

Reply:
Sorry, I'm confused, if I'm right, config options are not validated but you also say it should be safe to initialize storages because we can guarantee that their settings are valid according to declarations. So what will happen if e.g. `create_path` is not a boolean?

Reply:
I misread your question. In the first message you wrote

> validation happened a few lines before

That's true. Validation of all config options, **including options of storages**, happens a few lines before. That's why starting from line 155, we don't need additional explicit validation of storages  -they were already validated by this moment.

As a result, if you provide the wrong option, you'll never get to line 155, because of the ConfigurationException a few lines above.

Reply:
All clear, thanks. I found the comment confusing, what about something like:

```suggestion
    # Declared storage config options are valid at this point, storages can be configured
```